hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -47,7 +48,7 @@ def __init__(
         root_path: Path,
         batch_size: int = 1,
         seed: Optional[int] = None,
-        transform: Optional[Callable] = None,
+        transform: Optional[Dict[str, Callable]] = None,
```

Comment:
Also here it isn't clear from the variable name that it holds a dict with more than one transform

Reply:
changed